tools/update-fixtures.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+for i in $(./bin/test-libdrafter | grep Filename | sed -e 's/^  Filename: //' | sed -e 's/\x1B\[[0-9;]*[JKmsu]//g')
```

Comment:
You can already do `env GENERATE=1 make test` to update fixtures, was you unaware of that?

Reply:
Oh, I remember there were some discussion related it is not working.
So I probably miss info it is working again.

Ok, so i will remove this script.